cost-analyzer/values.yaml

```diff
@@ -304,6 +304,9 @@ global:
 ##
 upgrade:
   toV2: false
+  ## This flag is only required for users upgrading to 2.3 or later from a previous version.
+  ## This is because 2.3 will trigger a full re-ingestion of all available data into aggregator.
+  acknowledgeV2dot3FullReingestion: false
```

Comment:
I'd suggest making this value simpler. The whole matter of re-ingestion is an implementation detail. The user-facing point is just "get me to 2.3".